class.jetpack-modules-list-table.php

```diff
@@ -115,13 +115,15 @@ function get_views() {
 			'all' => sprintf( $format, $title, $count, $url, $current ),
 		);
 		foreach ( $module_tags_unique as $title => $count ) {
```

Comment:
@jessefriedman 

What I would rather see here is:

```if( 'Jumpstart' !== $title ) {
continue;
}```

Rather than wrapping up just the bottom section, this makes it much easier to see what's happening, and makes it less likely to bork in the future